lib/components/Properties/DateProperty.tsx

```diff
@@ -0,0 +1,52 @@
+import { CalendarDays } from 'lucide-react'
+import { tx } from '../../twind'
+import { formatDate } from '../../util/date'
+import type { PropertyBaseProps } from './PropertyBase'
+import { PropertyBase } from './PropertyBase'
+
+export type DatePropertyProps = Omit<PropertyBaseProps, 'icon' | 'input' | 'hasValue'> & {
+  date?: Date,
+  onChange?: (date: Date) => void
+}
+
+/**
+ * An Input for date properties
+ */
+export const DateProperty = ({
+  date,
+  onChange,
+  ...baseProps
+}: DatePropertyProps) => {
+  const hasValue = !!date
+
+  const dateText = date ? formatDate(date) : ''
+  return (
+    <PropertyBase
+      {...baseProps}
+      hasValue={hasValue}
+      icon={<CalendarDays size={16}/>}
+      input={({ required }) => (
+        <div
+          className={tx('flex flex-row grow py-2 px-4 cursor-pointer', { 'text-hw-warn-600': required && !hasValue })}
+        >
+          <input
```

Comment:
I think this is the second time we need a date input. The first time was in [TaskDetails](https://github.com/helpwave/web/blob/main/tasks/components/layout/TaskDetailView.tsx#L186). Please move this out into a separate component. Can we then also get rid of this calendar icon or make it optional?

Reply:
WIthout the icon only manual date input is possible and we would have to import or write our own datatime picker

Reply:
Will be done in #895